src/components/StatusUpdateForm.js

```diff
@@ -29,8 +29,8 @@ const ImageState = Object.freeze({
   UPLOAD_FAILED: Symbol('upload_failed'),
 });
 
-const urlRegex = /(https?:\/\/[^\s]+)/gi;
-
+const urlRegex = /(?:[\w/:@-]+\.[\w/:@.-]*)+(?=\s|$)/g;
+const emailRegex = /.*@.*/ ;
```

Comment:
In discussion with Anatoly, for better regex!! Will replace it!

Reply:
what if the url has an `@` in the query params for some reason?